.pre-commit-config.yaml

```diff
@@ -25,16 +25,19 @@ repos:
         args: ["--remove"]
       - id: mixed-line-ending
         args: ["--fix=lf"]
-  - repo: https://github.com/markdownlint/markdownlint
-    rev: v0.11.0
-    hooks:
-      - id: markdownlint
-        args: ["-r", "~MD002,~MD013,~MD029,~MD033,~MD034",
-              "-t", "allow_different_nesting"]
   - repo: https://github.com/codespell-project/codespell
     rev: v2.1.0
     hooks:
       - id: codespell
+  - repo: https://github.com/executablebooks/mdformat
+    rev: 0.7.14
+    hooks:
+      - id: mdformat
+        args: ["--number"]
+        additional_dependencies:
```

Comment:
some rules are skipped in the original markdownlint, should we also set some rules here to lose some restriction?

Reply:
`mdformat` does not accept those rules to skip checking. Maybe we need to discuss those rules.

Reply:
https://github.com/executablebooks/mdformat/issues/53